hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/ratis/conf/RatisClientConfig.java

```diff
@@ -203,6 +203,21 @@ public void setExponentialPolicyMaxSleep(Duration duration) {
     exponentialPolicyMaxSleep = duration;
   }
 
+  @Config(key = "client.exponential.backoff.max.retries",
+      defaultValue =  "2147483647",
```

Comment:
```suggestion
      defaultValue = String.valueOf(Integer.MAX_VALUE),
```

Reply:
`String.valueOf` doesn't return constant but `defaultValue` requires a constant String.
Other way we can change like `"" + Integer.MAX_VALUE` but this has overhead of `StringBuilder.`

Reply:
You could define 
```
  private final static string intMax = Integer.toString(Integer.MAX_VALUE);
``` 
in this class and then assign defaultValue = intMax?

Reply:
I tried this before. Tried again, but it doesn't work. Same error "Attribute value must be constant".